packages/console/src/pages/Cloud/pages/Welcome/index.tsx

```diff
@@ -83,7 +83,7 @@ const Welcome = () => {
           </form>
         </div>
       </OverlayScrollbar>
-      <ActionBar>
+      <ActionBar step={1}>
```

Comment:
I don't see a `step={3}` in this PR. Did you forget about it?

Reply:
🚧: LOG-5304